src/test/java/com/hotels/mutantswarm/model/MutantSwarmStatementTest.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.mutantswarm.model;
+
+import org.apache.hadoop.hive.ql.parse.MutantSwarmParseDriver;
+import org.junit.Test;
+
+import com.hotels.mutantswarm.model.MutantSwarmStatement.Factory;
+
+public class MutantSwarmStatementTest {
+
+  MutantSwarmParseDriver parseDriver = new MutantSwarmParseDriver();
+
+  private Factory factory = new Factory(parseDriver);
+
+  @Test(expected = RuntimeException.class)
+  public void checkNewInstance() {
+    MutantSwarmStatement mutantSwarmStatement = factory.newInstance(2, 3, "\"CREATE TABLE foobar AS\\nSELECT c\\nFROM bar\\nWHERE b = 3\"");
```

Comment:
why is this expected to throw an error? Might be good to also have a test which checks typical behaviour

Reply:
Because the query is written in an incorrect format; either way, it's a good idea to add a test for when it works correctly